Description: Upstream changes introduced in version 0.03
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 classicmenu-indicator (0.03) natty; urgency=low
 .
    * cmindicator.py:
      - create_menu_item: set always_show_image to True
      - create_indicator: use miono style icon (LP: #792780). Thanks to Sander van Noort
 .
 The person named in the Author field signed this changelog entry.
Author: Florian Diesch <debian@florian-diesch.de>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/792780

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- classicmenu-indicator-0.03.orig/Makefile
+++ classicmenu-indicator-0.03/Makefile
@@ -1,6 +1,6 @@
 NAME=classicmenu-indicator
-VERSION=0.02
-DEBVERSION=0.02
+VERSION=0.03
+DEBVERSION=0.03
 PPA=diesch/testing
 
 .PHONY: clean deb sdist ppa deb
